src/Bundle/test/src/Migration/SyliusAdminOrder.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+/**
+ * This code is generated by the config converter under https://github.com/mamazu/grid-config-converter
+ * Feel free to modify the code as you see fit.
+ */
+
+namespace SomeNameSpace;
+
+use Sylius\Bundle\GridBundle\Builder\Action\ShowAction;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\ItemActionGroup;
+use Sylius\Bundle\GridBundle\Builder\Field\DateTimeField;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\Field\TwigField;
+use Sylius\Bundle\GridBundle\Builder\Filter\Filter;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+class SyliusAdminOrder extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public static function getName(): string
+    {
+        return 'sylius_admin_order';
+    }
+
+    public function getResourceClass(): string
+    {
+        return 'Sylius\\Model\\Order';
```

Comment:
maybe we can detect if its is a FQCN here.

Reply:
Also done.